Equality/ViewModels/Projects/CreateColumnControlViewModel.cs

```diff
@@ -14,26 +14,75 @@
 using Equality.MVVM;
 using Equality.Models;
 using Equality.Services;
+using System.Windows.Input;
+using Equality.Data;
 
 namespace Equality.ViewModels
 {
     public class CreateColumnControlViewModel : ViewModel
     {
-        public CreateColumnControlViewModel()
+        IColumnService ColumnService;
+
+        public CreateColumnControlViewModel(IColumnService columnService)
         {
+            ColumnService = columnService;
+
+            CreateBoard = new TaskCommand<KeyEventArgs>(OnCreateBoardExecute);
+            CloseWindow = new TaskCommand(OnCloseWindowExecute);
         }
 
         #region Properties
 
         [Model]
-        public Team Team { get; set; } = new();
+        public Column Column { get; set; } = new();
 
+        [ViewModelToModel(nameof(Column))]
+        [Validatable]
         public string Name { get; set; }
 
         #endregion
 
         #region Commands
 
+        public TaskCommand<KeyEventArgs> CreateBoard { get; private set; }
+
+        private async Task OnCreateBoardExecute(KeyEventArgs args)
+        {
+            if (args != null) {
+                if (args.Key == Key.Escape) {
+                    CloseWindow.Execute();
+                }
+
+                if (args.Key != Key.Enter) {
+                    return;
+                }
+            }
```

Comment:
We dont need this (or make it work, cause right now it is not works).
If you delete this, also remember to delete `KeyEventArgs` from method and commands signatures.